lib/main_common.pm

```diff
@@ -1708,20 +1704,22 @@ sub load_extra_tests_docker {
     my ($image_names, $stable_names) = get_suse_container_urls();
     return unless @$image_names;
 
-    loadtest "console/zypper_ref";
+    if (is_repo_replacement_required && is_jeos) {
```

Comment:
Why `is_jeos`?

I don't see any reason for the `zypper_ref` anyway (the PR which introduced it didn't provide one either), so I'd just drop that.

That leaves us with just

`replace_opensuse_repos_tests if is_repo_replacement_required;`

Ideally that should happen before `load_extra_tests` in general, but for now it's ok

Reply:
> Why `is_jeos`?
> 

Haven't really checked the behavior outside JeOS. 

> I don't see any reason for the `zypper_ref` anyway (the PR which introduced it didn't provide one either), so I'd just drop that.
> 
> That leaves us with just
> 
> `replace_opensuse_repos_tests if is_repo_replacement_required;`
> 
> Ideally that should happen before `load_extra_tests` in general, but for now it's ok

Sure, let's do it the better way. I have found the function call [opensuse/main.pm#L419)](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/opensuse/main.pm#L419).
The easy way would be to put this check directly at the beginning of `load_extra_tests` and remove it from the `load_extra_tests_${component}` calls. WDYT ?